packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -35,7 +35,11 @@ export function DateTimePicker({
 }: DateTimePickerProps) {
   const {t} = useTranslation()
 
-  const [dateSelection, setDateSelection] = useState<any>(dateTime)
+  const [dateSelection, setDateSelection] = useState<Date | null>(dateTime ?? null)
+
+  useEffect(() => {
```

Comment:
Fixes bug that when dateTime is changed on the parent it will not update in the child